### PR TITLE
add charts to survey data

### PR DESCRIPTION
**Pull request by Mankavelda**
Used chart.js to add some charts on survey data.

The pie chart lists frameworks and percentage usage associated with each:

frameworks

The bar chart lists languages and the number of people using the language:

languages

### CODE REVIEW

**Comment by Mankavelda** [comment]
Used charts.js to add a pie chart and a bar chart in the index file.
The pie charts shows the percentage usage of various frameworks, while the bar chart shows programming languages and number of people using it.

File: src/views/index.hbs [hunk 1]
```diff
@@ -6,11 +6,169 @@
...
+
+    <!--pie chart-->
+  
+     <div style="margin-left:5%;margin-right:5%">
+        <canvas id="myChart" style="width:100%;max-width:80%;"></canvas>
+    </div>
+
+  <script>
+  var xValues = ["Flask", "React.js", "Unity", "USP.NET Core", "Laravel", "Drupal", "OWL(Odoo Web Library)", "Odoo", "Spring", "jQuery", "Vue.js", "Angular.js", "Angular", "Express", "FastAPI", "Django"];
+  var yValues = [19, 7, 4, 4, 4, 4, 4, 4, 4, 4, 7, 7, 11, 4, 7, 7,];
```
Comment (elhmn) [inline src/views/index.hbs:25]: I will suggest you use `let` everywhere you have used `var` unless you are dealing with a variable that will not be modified then use `const`.
```suggestion
  let yValues = [19, 7, 4, 4, 4, 4, 4, 4, 4, 4, 7, 7, 11, 4, 7, 7,];
```

File: src/views/index.hbs [hunk 2]
```diff
@@ -6,11 +6,169 @@
...
+    
+
+
+    <!--pie chart-->
+  
+     <div style="margin-left:5%;margin-right:5%">
+        <canvas id="myChart" style="width:100%;max-width:80%;"></canvas>
+    </div>
+
+  <script>
```
Comment (elhmn) [inline src/views/index.hbs:23]: Your scripts should be defined in a separate js file and imported here via `<script ... url=your_script_file >`

**Review by elhmn — CHANGES_REQUESTED** [review]

File: src/views/index.hbs (hunk 1 above)
Comment (Mankavelda) [reply src/views/index.hbs:25]: Noted

File: src/views/pie.js [hunk 3]
```diff
@@ -0,0 +1,66 @@
+  let pieChart = document.getElementById("myChart");
+  let xValues = ["Flask", "React.js", "Unity", "USP.NET Core", "Laravel", "Drupal", "OWL(Odoo Web Library)", "Odoo", "Spring", "jQuery", "Vue.js", "Angular.js", "Angular", "Express", "FastAPI", "Django"];
+  let yValues = [19, 7, 4, 4, 4, 4, 4, 4, 4, 4, 7, 7, 11, 4, 7, 7,];
```
Comment (elhmn) [inline src/views/pie.js:3]: Now this data should be coming from the [survey.yaml](https://github.com/osscameroon/devsurvey/blob/main/src/res/survey.yaml) file. We should not write them ourselves.

File: src/views/bar.js [hunk 4]
```diff
@@ -0,0 +1,59 @@
+//bar chart
+
+let barChart = document.getElementById("myBarChart")
+let xVal = ["swift", "C/C++", "C#", "Golang", "Rust", "Python", "javascript", "typescript", "PHP", "Bash", "Go", "Groovy", "VBA", "dart", "Ada", "Scalar", "Java"];
```
Comment (elhmn) [inline src/views/bar.js:4]: Now this data should be coming from the [survey.yaml](https://github.com/osscameroon/devsurvey/blob/main/src/res/survey.yaml) file. We should not write them ourselves.

**Review by elhmn — COMMENTED** [review]
Nice work so far, now let's make sure we get the data from the actual survey.yaml file

**Pull request closed without merging**